src/runtime/opencl/opencl_device_api.cc

```diff
@@ -72,20 +75,26 @@ void OpenCLWorkspace::GetAttr(Device dev, DeviceAttrKind kind, TVMRetValue* rv)
       *rv = static_cast<int64_t>(value);
       break;
     }
-    case kComputeVersion:
-      return;
-    case kDeviceName: {
-      char value[128] = {0};
-      OPENCL_CALL(
-          clGetDeviceInfo(devices[index], CL_DEVICE_NAME, sizeof(value) - 1, value, nullptr));
-      *rv = std::string(value);
+    case kComputeVersion: {
+      std::string ret = GetDeviceInfo(devices[index], CL_DEVICE_VERSION);
+      // String returned is "OpenCL $MAJOR.$MINOR $VENDOR_INFO".  To
+      // match other implementations, we want to return "$MAJOR.$MINOR"
+      const size_t version_start = 7;
```

Comment:
why is it 7?

Reply:
7 characters skips past the initial "OpenCL " at the start of the version returned by OpenCL.  (e.g. "OpenCL 1.2 CUDA" is returned on my machine).  I can make it more explicit as `strlen("OpenCL ")` instead.

Reply:
ok cool, yeah either that or just add comment would be great!